pkg/parca/parca.go

```diff
@@ -433,6 +432,11 @@ func Run(ctx context.Context, logger log.Logger, reg *prometheus.Registry, flags
 			if err != nil && !errors.Is(err, context.Canceled) {
 				level.Error(logger).Log("msg", "error shutting down server", "err", err)
 			}
+
+			// Close the columnstore after the parcaserver has shutdown to ensure no more writes occur against it.
+			if err := col.Close(); err != nil {
```

Comment:
So this is the crux of this PR, is it? Was looking for a bit to find it. The close function itself is defined in FrostDB? 
Very nice to tie it into the run.Group shutdown too. :+1: